src/Result.php

```diff
@@ -16,13 +16,23 @@ public function isValid(): bool
         return $this->errors === [];
     }
 
-    public function addError(string $message): void
+    /**
+     * @param string $message
```

Comment:
```suggestion
     * Add an error.
     *
     * @param string $message Error message.
```

Reply:
It's literally just a method name. Maybe extend it a bit with details or do not add at all yet? And it will be inconsistent with other missing descriptions, for example, for `isValid()` method, etc.

Reply:
Or maybe there are some other documentation nuances involved here?